lsp/io.ml

```diff
@@ -61,9 +61,9 @@ let log = ref (fun _ _ -> ())
 let send_json fmt obj =
   Mutex.lock mut;
   if Fleche.Debug.send then !log "send" obj;
-  let msg = F.asprintf "%a" J.(pretty_print ~std:true) obj in
+  let msg = J.to_string obj in
```

Comment:
Nice catch!

Let's keep the pretty printed version around if possible for a while (disabled under a flag), it is rare, but sometimes we need to do wire-level debugging and in that case the pretty printing helps.

Reply:
Yeah, that makes sense. Once I understand how to use Fleche.Config it shouldn't be hard to add an extra parameter for json pretty printing, I was just cautious when I thought it meant threading another parameter through a bunch of function calls.